lib/config/presets/internal/replacements.ts

```diff
@@ -0,0 +1,17 @@
+import type { Preset } from '../types';
+
+export const presets: Record<string, Preset> = {
+  replacements: {
```

Comment:
If it's just one rule, do we necessarily need a new preset category/namespace for it? And is it `replacements:replacements`?

Reply:
I thought that this could be expanded on by the community? I am taking suggestions for better names 😅